internal/util/workspace.go

```diff
@@ -6,13 +6,24 @@ package util
 import (
 	"fmt"
 	"net/url"
+	"os"
 	"regexp"
 	"strings"
 )
 
-func GetValidatedWorkspaceName(input string) (string, error) {
-	// input = strings.ToLower(input)
+func WorkspaceMode() bool {
```

Comment:
Why the move?

Reply:
Looks like a duplicate actually, probably a rebasing mistake, good catch